docs/products/tools/marketplace/guides/secure-your-server/index.md

```diff
@@ -0,0 +1,59 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: "Automate common security tasks on a Linode Compute Instance by using the Secure Your Server Marketplace App."
+keywords: ['security']
+tags: ["marketplace", "linode platform", "cloud manager"]
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2023-01-11
+title: "Secure Your Server through the Linode Marketplace"
+contributor:
+  name: Linode
+---
+
+This Marketplace App automatically configures a new Compute Instance with a limited user account and other best practices discussed in the [Setting Up and Securing a Compute Instance](/docs/guides/set-up-and-secure/) guide. In addition to creating a limited user with sudo access, the app updates the system, optionally configures a domain in the DNS Manager, and optionally configures a Block Storage Volume. It also configures a basic firewall through [UFW](/docs/guides/configure-firewall-with-ufw/) and enables [Fail2Ban](/docs/guides/using-fail2ban-to-secure-your-server-a-tutorial/).
+
+## Deploying a Marketplace App
+
+{{< content "deploy-marketplace-apps-shortguide">}}
+
+{{< content "marketplace-verify-standard-shortguide">}}
+
+{{< note >}}
+**Estimated deployment time:** The Secure Your Server tasks should be fully installed within 2-5 minutes after the Compute Instance has finished provisioning.
+{{< /note >}}
+
+## Configuration Options
+
+- **Supported distributions:** Ubuntu 20.04 LTS, Debian 10, Debian 11
+- **Recommended plan:** All plan types and sizes can be used.
+
+### Secure Your Server Options
+
+#### Limited User (Required)
+
+Fill out the following fields to automatically create a limited user for your new Compute Instance. This account will be assigned to the *sudo* group, which provides elevated permission when running commands with the `sudo` prefix.
+
+- **Limited sudo user:** Enter your preferred username for the limited user.
+- **Password for the limited user:** Enter a *strong* password for the new user.
+- **SSH public key for the limited user:** If you wish to login as the limited user through public key authentication (without entering a password), enter your public key here. See [Creating an SSH Key Pair and Configuring Public Key Authentication on a Server](/docs/guides/use-public-key-authentication-with-ssh/) for instructions on generating a key pair.
+- **Disable root access over SSH:** To block the root user from logging in over SSH, select *Yes* (recommended). You can still switch to the root user once logged in and you can also log in as root through [Lish](/docs/guides/lish/).
+
+{{< content "marketplace-custom-domain-fields-shortguide">}}
+
+- **Email address:** The start of authority (SOA) email address for this server. This is a required field if you want the installer to create DNS records.
+- **Send email from domain:** If you'd like to be able to send email from the custom domain you optionally configured, select **Yes**.
```

Comment:
This might just be me, but it feels like this line suggests that the marketplace app will install email software onto the Linode (I assume it doesn't)

Reply:
Added more information to address this.